rest/src/main/java/com/redhat/ipaas/rest/v1/handler/connection/ConnectorHandler.java

```diff
@@ -38,4 +39,9 @@ public String resourceKind() {
         return Connector.KIND;
     }
 
+    @Path("/{id}/actions")
+    public ConnectorActionHandler getActions(@PathParam("id") String connectorId) {
```

Comment:
Magic :) Honestly I don't know _how_ it works, I only learned this morning that this is the way to do it.